lib/signaling/v2/room.js

```diff
@@ -94,16 +94,16 @@ class RoomV2 extends RoomSignaling {
       _RemoteParticipantSignaling: {
         value: options.RemoteParticipantSignaling
       },
-      _subscribed: {
-        value: new Map()
-      },
       _subscribedRevision: {
         value: 0,
         writable: true
       },
       _subscriptionFailures: {
         value: new Map()
       },
+      _trackSidsToTrackIds: {
```

Comment:
Is this a map of track Sid and Mid? If so can we make it more explicit?

Reply:
@dipankadas It is a Map of Track SIDs to Track IDs.